components/style/properties/longhand/font.mako.rs

```diff
@@ -475,3 +475,48 @@ ${helpers.single_keyword("font-variant-position",
         }
     }
 </%helpers:longhand>
+
+<%helpers:longhand name="font-language-override" products="none" animatable="False">
```

Comment:
Could you add a link to the spec for this here please? https://www.w3.org/TR/css-fonts-3/#propdef-font-language-override would do :)